### ISSUE TITLE
IPv6 issue

### ISSUE THREAD

**Issue by johncarterofmars** [getumbrel/umbrel#1552]
I am running Umbrl on Ubuntu 22.04. I have disabled IPv6 for this host. Everything is working properly except for Simple Torrent and Synapse. 
Simple Torrent:
`Attaching to simple-torrent_app_proxy_1, simple-torrent_server_1
app_proxy_1  | yarn run v1.22.19
app_proxy_1  | $ node ./bin/www
app_proxy_1  | [HPM] Proxy created: /  -> http://10.21.21.41:8086
app_proxy_1  | Waiting for 10.21.21.41:8086 to open...
server_1     | 2022/11/13 22:13:30 [engine][Configure] error subsequent listen: listen tcp6 :50007: socket: address family not supported by protocol
server_1     | 2022/11/13 22:13:33 [engine][Configure] error subsequent listen: listen tcp6 :50007: socket: address family not supported by protocol
server_1     | 2022/11/13 22:13:36 [engine][Configure] error subsequent listen: listen tcp6 :50007: socket: address family not supported by protocol
server_1     | 2022/11/13 22:13:39 [engine][Configure] error subsequent listen: listen tcp6 :50007: socket: address family not supported by protocol
server_1     | 2022/11/13 22:13:42 subsequent listen: listen tcp6 :50007: socket: address family not supported by protocol
server_1     | 2022/11/13 22:13:43 ############# SimpleTorrent ver[1.3.9] #############
server_1     | 2022/11/13 22:13:43 [server]WARNING: --host --port arguments are depreciated, use --linsten instead, eg:`--listen :3000`
server_1     | 2022/11/13 22:13:43 [engine][config] using config file:  /config/simple-torrent.json
server_1     | 2022/11/13 22:13:43 [engine][Configure] 64bit arch detected, using MMap for storage
server_1     | 2022/11/13 22:13:43 [engine][Configure] error subsequent listen: listen tcp6 :50007: socket: address family not supported by protocol`

Synapse:
`Attaching to synapse_server_1, synapse_app_proxy_1
app_proxy_1  | Error wating for port: "connect EHOSTUNREACH 10.21.21.44:8008"
app_proxy_1  | Retrying...
app_proxy_1  | Error wating for port: "connect EHOSTUNREACH 10.21.21.44:8008"
app_proxy_1  | Retrying...
app_proxy_1  | Error wating for port: "connect EHOSTUNREACH 10.21.21.44:8008"
app_proxy_1  | Retrying...
app_proxy_1  | Error wating for port: "connect EHOSTUNREACH 10.21.21.44:8008"
app_proxy_1  | Retrying...
app_proxy_1  | Error wating for port: "connect EHOSTUNREACH 10.21.21.44:8008"
app_proxy_1  | Retrying...
server_1     |     check_bind_error(e, address, bind_addresses)
server_1     |   File "/usr/local/lib/python3.8/site-packages/synapse/app/__init__.py", line 50, in check_bind_error
server_1     |     raise e
server_1     |   File "/usr/local/lib/python3.8/site-packages/synapse/app/_base.py", line 261, in listen_tcp
server_1     |     r.append(reactor.listenTCP(port, factory, backlog, address))
server_1     |   File "/usr/local/lib/python3.8/site-packages/twisted/internet/posixbase.py", line 568, in listenTCP
server_1     |     p.startListening()
server_1     |   File "/usr/local/lib/python3.8/site-packages/twisted/internet/tcp.py", line 1335, in startListening
server_1     |     raise CannotListenError(self.interface, self.port, le)
server_1     | twisted.internet.error.CannotListenError: Couldn't listen on :::8008: [Errno 97] Address family not supported by protocol.
`

Is there a way to disable IPv6 for these 2 so that they can start?
Thank you.

**Comment by superswan** [comment]
Is there a reason you have ipv6 disabled on the host? It is well known disabling ipv6 on Linux can cause problems. I don't use either of these apps and I can't tell if the issue is with the app itself or the app proxy from the submitted info but the problem appears to be with `Couldn't listen on :::8008: [Errno 97] Address family not supported by protocol.` I would look at the docker file for these apps and if anywhere the listening address is defined with `::8008` change to `0.0.0.0:8008` 

Basically it looks the problem is with the app listening on ipv6 interface (which is disabled)
